express-get-json/index.js

```diff
@@ -0,0 +1,34 @@
+const express = require('express');
+const app = express();
+
+const grades = {
+  12: {
+    id: 12,
+    name: 'Tim Berners-Lee',
+    course: 'HTML',
+    score: 95
+  },
+  47: {
+    id: 47,
+    name: 'Brendan Eich',
+    course: 'JavaScript',
+    score: 100
+  },
+  273: {
+    id: 273,
+    name: 'Forbes Lindsay',
+    course: 'JavaScript',
+    score: 92
+  }
+};
+
+app.get('/api/grades', (req, res) => {
+  for (const property in grades) {
+    console.log(`${property}: ${grades[property]}`);
+  }
+  res.json({ grades });
```

Comment:
Here you want to create an array, loop through the `grades` object, push each grade into the array, then send the array back to the client.
```JAVASCRIPT
const gradesArray = [];
for (const id in grades) {
  gradesArray.push(grades[id]);
}
res.json(gradesArray);
 ```